routes/web.php

```diff
@@ -87,6 +87,14 @@
     return redirect("/build/{$buildid}/notes", 301);
 });
 
+Route::get('/build/{id}/files', 'BuildController@files');
```

Comment:
The route parameter needs to match the name of the parameter on the corresponding function for Laravel to automatically do the type conversion.